Belot/Belot.Server/SignalR/BelotHub.cs

```diff
@@ -1,18 +1,19 @@
-﻿using Belot.Data;
-using Belot.Models.Belot;
-using Belot.Models.DataEntries;
-using Belot.Models.Http.Requests.SignalR;
-using Belot.Models.Http.Responses.SignalR;
-using Belot.Services.Application;
-using Belot.Services.Belot;
-using Belot.Services.Interfaces;
-using Belot.Services.Logging;
-using Belot.Services.Logging.Interfaces;
-using Microsoft.AspNetCore.SignalR;
-using System.Diagnostics;
-
-namespace Belot.SignalR
+﻿namespace Belot.SignalR
 {
+    using Belot.Data;
```

Comment:
What's the purpose of in-namespace usings?

Reply:
On most places I think I added usings to be inside namespaces. This is based on most StyleCop rules, but technically does not matter where they should be. It's just a matter of taste. I stick to this way because on most of my personal projects I am using StyleCop rules.